.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,10 +1,12 @@
 ---
 name: Custom issue template
-about: Describe this issue template's purpose here.
+about: Issue template for custom issues.
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
+***NOTE:*** *Please ask your "how to" questions or feature existence queries on the [stackoverflow](https://stackoverflow.com/questions/tagged/ant-media-server) or [our community forum](https://groups.google.com/forum/#!forum/ant-media-server).*
 
+***NOTE:*** *If you are sure you found a bug, please fill out the bug report form abd create a [bug issue](https://github.com/ant-media/Ant-Media-Server/issues/new?assignees=&labels=&template=bug_report.md&title=).*
```

Comment:
I think this information is redundant because user already choose the template before coming here.

Reply:
removed